helm_deploy/hmpps-restricted-patients/Chart.yaml

```diff
@@ -5,7 +5,7 @@ name: hmpps-restricted-patients
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 2.6.5
+    version: "2.7"
```

Comment:
odd to be in quotes - also don't think we normally float on the minor version do we?

Reply:
you mean the patch version?  this is really individual preference. Personally I prefer to bring in patch versions automatically.

Reply:
It's quoted as 2.7 is a number in yaml where as "2.6.5" is a string

Reply:
okay thanks for both answers.  Will change all of ours to be floating on patch...